src/main/java/com/krafit/financialcalcs/LoanCalculatorApplication/LoanCalculator.java

```diff
@@ -0,0 +1,14 @@
+package com.krafit.financialcalcs.LoanCalculatorApplication;
```

Comment:
ExpectedPersonalLoanEMICalculationRequest

Reply:
changed to ExpectedPersonalLoanEMICalculationRequest